Source/JNA/waffle-tomcat8/src/main/java/waffle/apache/NegotiateAuthenticator.java

```diff
@@ -158,12 +159,11 @@ public boolean authenticate(final Request request, final HttpServletResponse res
             try {
                 this.log.debug("logged in user: {} ({})", windowsIdentity.getFqn(), windowsIdentity.getSidString());
 
-                final GenericWindowsPrincipal windowsPrincipal = new GenericWindowsPrincipal(windowsIdentity,
-                        this.principalFormat, this.roleFormat);
+                final GenericPrincipal genericPrincipal = createPrincipal(windowsIdentity);
 
-                this.log.debug("roles: {}", windowsPrincipal.getRolesString());
+                this.log.debug("roles: {}",  String.join(", ", genericPrincipal.getRoles()));
```

Comment:
Minor: one space too many.

Reply:
Run a quick build, that will flush out that space or any others.

Reply:
@Snap252 There isn't really much difference between the various integrations.  We probably could use an overall refactoring to make things better.  For now though, as new builds become available, because we brand the version, we supply a new copy.  So tomcat 9 is really just tomcat 8.5 with the minor variance you saw.  Spring 3, 4, and 5 copies I believe are completely identical.  Just haven't had time to really go in and look how to make it more streamlined.

Reply:
> Run a quick build, that will flush out that space or any others.
> - hazendaz

I already tried a maven test / maven install - but this kills me all the formatting (in existing and untouched files) - Am I doing something wrong, or is there anything I needed to know to keep formatting?